.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+  - 3.5
+
+install:
+  - pip install Django
```

Comment:
If we end up supporting multiple versions of django, shouldn't we use environment variables?
[python docs](https://docs.travis-ci.com/user/languages/python/)